core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1067,9 +1067,9 @@ private[spark] object Utils extends Logging {
     // finding the call site of a method.
     val SPARK_CORE_CLASS_REGEX =
       """^org\.apache\.spark(\.api\.java)?(\.util)?(\.rdd)?(\.broadcast)?\.[A-Z]""".r
-    val SCALA_CLASS_REGEX = """^scala""".r
+    val SCALA_CLASS = "scala"
```

Comment:
Yes I understand that effect, but it's a trivial condition to check, so don't think it overcomes the overhead of the generated code to implement "lazy". It doesn't matter; it's merged.